htdocs/societe/class/societe.class.php

```diff
@@ -4410,7 +4403,7 @@ public function setMysoc(Conf $conf)
 		$this->logo_squarred_mini = getDolGlobalString('MAIN_INFO_SOCIETE_LOGO_SQUARRED_MINI');
 
 		// Define if company use vat or not
-		$this->tva_assuj = getDolGlobalString('FACTURE_TVAOPTION');
+		$this->tva_assuj = (is_numeric(getDolGlobalString('FACTURE_TVAOPTION'))) ? (int) getDolGlobalString('FACTURE_TVAOPTION') : 0;
```

Comment:
I think this option FACTURE_TVAOPTION is 0 or 1 so you may replace with
`$this->tva_assuj = getDolGlobalInt('FACTURE_TVAOPTION');`

Reply:
I checked and FACTURE_TVAOPTION is effectively only 0 or 1. Thanks.